src/tree/updater_quantile_hist.cc

```diff
@@ -259,9 +263,11 @@ void QuantileHistMaker::Builder::ExpandWithLossGuide(
 
   this->InitNewNode(0, gmat, gpair_h, *p_fmat, *p_tree);
 
-  this->EvaluateSplit(0, gmat, hist_, *p_fmat, *p_tree);
-  qexpand_loss_guided_->push(ExpandEntry(0, p_tree->GetDepth(0),
-                                         snode_[0].best.loss_chg, timestamp++));
+  ExpandEntry node(0, p_tree->GetDepth(0), snode_[0].best.loss_chg, timestamp++);
```

Comment:
The depth of root node is determined I believe.

Reply:
Changed 0 to ExpandEntry::kRootNid const, if I correctly understand the comment.